src/lib/serialization/serializers/comments/comment-tag.ts

```diff
@@ -0,0 +1,39 @@
+import { Component } from '../../../utils/component';
+import { CommentTag } from '../../../models';
+
+import { SerializerComponent } from '../../components';
+
+@Component({name: 'serializer:comment-tag'})
+export class CommentTagSerializer extends SerializerComponent<CommentTag> {
+
+  static PRIORITY = 1000;
+
+  /**
+   * Filter for instances of [[CommentTag]]
+   */
+  protected static serializeGroup(instance: any): boolean {
+    return instance instanceof CommentTag;
+  }
+
+  // use same fn for every instance
+  serializeGroup = CommentTagSerializer.serializeGroup;
+  serializeGroupSymbol = CommentTag;
+
+  initialize(): void {
+    super.initialize();
+    this.supports = (r: CommentTag) => true;
```

Comment:
purely speaking, this can be done in the prototype as a method and not a property, but it's not that important in this context because it's a plugin and only get's a instanciated once throughout the whole app.

I think the reason I did it that way is because it's done like that in other plugins I looked at to learn from.